test_apps/smokeTests/gridTools.cpp

```diff
@@ -334,6 +360,23 @@ bool RunTest(Grid *grid, bool silenceTime)
 
     PrintStats(rms, numMissingValues, disagreements, time, silenceTime);
 
+    // Test OpenMP implementation of Grid::GetRange()
+    float range_36[2] = {0.0, 1.1};
```

Comment:
Why the funny initialization values (0.0, 1.1, 2.2,etc.)?

Reply:
The goal of the correctness test is to compare equality. It helps to initialize them to different values, so that they won't be equal by accident.